src/js/components/AppListComponent.jsx

```diff
@@ -168,7 +168,7 @@ var AppListComponent = React.createClass({
   },
 
   hasFilters: function () {
-    return Object.values(this.props.filters).some(filter => {
+    return Object.values(AppsFiltersStore.filters).some(filter => {
```

Comment:
Think about the filterQueryParams as the "store".
`getQueryParamValue(FilterTypes.TEXT)`